helmclient.go

```diff
@@ -270,7 +338,7 @@ func (c *Client) InstallTiller() error {
 
 			return nil
 		}
-		b := newExponentialBackoff(2 * time.Minute)
+		b := newExponentialBackoff(2*time.Minute, 5*time.Second)
```

Comment:
We are using the default `maxInterval` everywhere but here because the tiller installation backoff is about checking the service is up while the other backoffs are about retrying on network errors. This is why this got configurable.